qa/zcash/postponed-updates.txt

```diff
@@ -60,9 +69,12 @@ native_rust 1.85.1 2025-12-31
 native_rust 1.86.0 2025-12-31
 native_rust 1.87.0 2025-12-31
 native_rust 1.88.0 2025-12-31
+native_rust 1.89.0 2025-12-31
+native_rust 1.90.0 2025-12-31
 
 # We follow upstream Bitcoin Core's LevelDB updates
 leveldb 1.23 2025-12-31
+leveldb 1.23 2025-12-31
```

Comment:
I can't tell with reasonable effort whether this release fixed any security issues; the changelog is too vague. In any case this was already postponed so the line is redundant.